toolchain/check/generic.h

```diff
@@ -46,6 +46,12 @@ auto MakeGenericInstance(Context& context, SemIR::GenericId generic_id,
 auto MakeGenericSelfInstance(Context& context, SemIR::GenericId generic_id)
     -> SemIR::GenericInstanceId;
 
+// Attempts to resolve the definition of the given specific, by evaluating the
+// eval block of the corresponding generic. Returns false if a definition is
+// not available.
```

Comment:
As part of resolving, does this modify anything? That may be important to comment on -- I don't quite know what "resolve" means in this context.

Reply:
Attempted to clarify.